pkg/operation/botanist/component/etcd/monitoring.go

```diff
@@ -406,13 +401,27 @@ func (e *etcd) AlertingRules() (map[string]string, error) {
 		return nil, err
 	}
 
+	// etcdQuorumReplicas := 1
+	// if e.hasHAControlPlane() {
+	// 	etcdQuorumReplicas = 2
+	// }
+	// x := *e.replicas
+	// quorum := int(*e.replicas/2) + 1
+	// fmt.Println("------------------------------------------------- ", x, quorum, x > 1)
```

Comment:
Nit: Is this block of commented code needed?

Reply:
Oops, had added this for some quick local testing. My apologies 😅